auth-web/src/util/common-util.ts

```diff
@@ -268,4 +268,9 @@ export default class CommonUtils {
   static trimTrailingSlashURL (url) {
     return (url) ? url.trim().replace(/\/+$/, '') : ''
   }
+
+  /** use entity type to get NR type from enum */
+  static mapRequestTypeCdToNrType (requestTypeCd: string): string {
```

Comment:
Update comment.

Reply:
Also, requestTypeCd has a type (not string).

Reply:
Comment is still incorrect.

Reply:
This comment was changed to "use NR request type code to get NR type from enum" a few commits ago. My latest commit fixes the comment for the nameRequestType function in affiliations-factory.ts